refinery/data_set_manager/utils.py

```diff
@@ -951,13 +950,13 @@ def get_file_url_from_node_uuid(node_uuid, require_valid_url=False):
             return core.utils.build_absolute_url(url) if url else None
         except ValueError as e:
             logger.error('URL {} is not a valid relative url'.format(str(url)))
-            raise type(e)(e.message)
+            raise type(e)(str(e))
         except RuntimeError as e:
             logger.error('Could not build absolute URL for {}'.format(
                     str(url)
                 )
             )
-            raise type(e)(e.message)
+            raise type(e)(str(e))
```

Comment:
Simply `raise`?

Reply:
I wanted to keep the message

Reply:
The message is kept as part of the exception that is re-raised:
```python
>>> try:
...     raise RuntimeError('blah')
... except RuntimeError:
...     raise
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
RuntimeError: blah
```